src/components/Menus/Sidebar/InstructionsPanel/InstructionsPanel.test.js

```diff
@@ -37,6 +37,14 @@ describe("When instructionsEditable is true", () => {
       );
     });
 
+    test("Renders two tab titles", () => {
+      expect(screen.getAllByRole("tab")).toHaveLength(2);
+    });
+
+    test("Renders two tab panels", () => {
+      expect(screen.getAllByRole("tabpanel")).toHaveLength(2);
+    });
+
```

Comment:
I think it would be good to replicate this below for when the instructions aren't editable